ExRouter.js

```diff
@@ -30,6 +30,7 @@ export class ExRouteAdapter {
         }
         this.route = route;
         this.name = route.name;
+        if (props.title) this.route.title = props.title;
```

Comment:
yes but this.title is on the `ExRouteAdapter` but the renderTitle callback is bound to `this.route`.